src/Microsoft.AspNet.Identity.EntityFramework/IdentityDbContext.cs

```diff
@@ -33,9 +33,14 @@ protected override void OnModelCreating(ModelBuilder builder)
             builder.Entity<TUser>(b =>
                 {
                     b.Key(u => u.Id);
+                    b.Index("NormalizedUserName", "NormalizedEmail");
```

Comment:
I think the two should be separate indexes unless we only have a query that searches for both together.